### PR TITLE
Update the changelog and bump version for releasing v0.4.0

### PR DESCRIPTION
**Pull request by yutannihilation**
https://github.com/extendr/libR-sys/issues/115

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## libR-sys (unreleased)
+## libR-sys 0.4.0
+
+- Update the default precomuted bindings to R 4.2.0.
+- [`Windows`] Dropped build-time dependency on `winapi`.
+- Remove bindings for the symbols that are not the part of R's API. [[#96]](https://github.com/extendr/libR-sys/pull/96)
```
Comment (Ilia-Kosenkov) [inline CHANGELOG.md:7]:
```suggestion
- Remove bindings for the symbols that are not part of R API. [[#96]](https://github.com/extendr/libR-sys/pull/96)
```

Comment (yutannihilation) [reply CHANGELOG.md:7]: Thanks

**Review by multimeric — APPROVED** [review]
What about:

* MSVC toolchain: https://github.com/extendr/libR-sys/pull/111
* RTools 43: https://github.com/extendr/libR-sys/pull/120

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+## libR-sys 0.4.0
+
+- Update the default precomuted bindings to R 4.2.0.
+- [`Windows`] Dropped build-time dependency on `winapi`.
+- Remove bindings for the symbols that are not part of R API. [[#96]](https://github.com/extendr/libR-sys/pull/96)
+- Add bindings for the following header files:
+  - `R_ext/Applic.h` [[#117]](https://github.com/extendr/libR-sys/pull/117)
+  - `R_ext/Random.h` [[#123]](https://github.com/extendr/libR-sys/pull/123)
+  - `Rmath.h` [[#124]](https://github.com/extendr/libR-sys/pull/124)
+- [`Linux`] Provide precomuted bindings for linux-aarch64. [[#133]](https://github.com/extendr/libR-sys/pull/133)
```
Comment (multimeric) [inline CHANGELOG.md:12]:
```suggestion
- [`Linux`] Provide precomuted bindings for linux-aarch64 (aka ARM64). [[#133]](https://github.com/extendr/libR-sys/pull/133)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
 # Changelog
 
-## libR-sys (unreleased)
+## libR-sys 0.4.0
+
+- Update the default precomuted bindings to R 4.2.0.
+- [`Windows`] Dropped build-time dependency on `winapi`.
+- Remove bindings for the symbols that are not part of R API. [[#96]](https://github.com/extendr/libR-sys/pull/96)
+- Add bindings for the following header files:
+  - `R_ext/Applic.h` [[#117]](https://github.com/extendr/libR-sys/pull/117)
```
Comment (multimeric) [inline CHANGELOG.md:9]:
```suggestion
  - `R_ext/Applic.h`: optimisation functions [[#117]](https://github.com/extendr/libR-sys/pull/117)
```

Is this a correct summary?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+  - `R_ext/Random.h` [[#123]](https://github.com/extendr/libR-sys/pull/123)
```
Comment (multimeric) [inline CHANGELOG.md:10]:
```suggestion
  - `R_ext/Random.h`: random number generator state wrappers [[#123]](https://github.com/extendr/libR-sys/pull/123)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,17 @@
...
+  - `Rmath.h` [[#124]](https://github.com/extendr/libR-sys/pull/124)
```
Comment (multimeric) [inline CHANGELOG.md:11]:
```suggestion
  - `Rmath.h`: distribution functions [[#124]](https://github.com/extendr/libR-sys/pull/124)
```

**Comment by yutannihilation** [comment]
Thanks.

> - MSVC toolchain: https://github.com/extendr/libR-sys/pull/111

We didn't announce we had changed the toolchain to the GNU toolchain, so this is not a news.

> - RTools 43: https://github.com/extendr/libR-sys/pull/120

As you can see, this was just about the CI and README, so not a news.

**Comment by yutannihilation** [comment]
As probably there's no big concern on this pull request, I'm merging. But, feel free to add your comment on #115 or create pull requests if you feel there's still anything that has to be done before releasing.

**Pull request merged**